### ISSUE TITLE
Make navigation from diff view to editor more discoverable

### ISSUE THREAD

**Issue by jcansdale** [github/VisualStudio#1546]
- GitHub Extension for Visual Studio version: 2.4.4

At the moment navigating from the diff view to the editor isn't very discoverable. The user needs to have checked out out a PR and already know they can navigate by hitting `Enter` or right-clicking and selecting the the `Open File in Solution` button.

Educating users using an info banner is one option, but getting the balance between attracting attention without being intrusive can be a challenge. It would be better if we could give the user a non-intrusive reminder at the moment they might want to use it.

We could display, `Press Enter to navigate to solution file` if/when the user clicks on the diff view. This would be an action the user could take immediately that wouldn't have worked before they clicked. The status message would be a response to a user action and immediately actionable.

**Comment by sguthals** [comment]
@jcansdale - I like this idea. Where do you think the message would be displayed?

**Comment by jcansdale** [comment]
@sguthals Oops, I missed half a sentence (updated). It would be on the status bar. I think that's a good place for transient messages that aren't intrusive like an info banner.

**Comment by sguthals** [comment]
Yeah, while I was reading this for some reason it popped into my head a neat little popup that came up that was super subtle...that might be too intrusive and distracting though after they know that the feature exists...
